CRM/ManualDirectDebit/Upgrader.php

```diff
@@ -7,11 +8,124 @@
 class CRM_ManualDirectDebit_Upgrader extends CRM_ManualDirectDebit_Upgrader_Base {
 
   public function install() {
+
     $this->executeSqlFile('sql/install.sql');
+    $this->createDirectDebitPaymentProcessorType();
```

Comment:
I also remember that I mentioned this before, but you can just rename sql/install.sql to sql/auto_install.sql and sql/uninstall.sql to sql/auto_uninstall.sql and you don't need these two lines any more : 

```php
 $this->executeSqlFile('sql/install.sql');
```

```php
$this->executeSqlFile('sql/uninstall.sql');
```